terraform/ecs-service/iam.tf

```diff
@@ -201,3 +229,156 @@ data "aws_iam_policy_document" "execute_command_policy" {
     ]
   }
 }
+
+resource "aws_iam_policy" "custom_addons_policy" {
```

Comment:
I wonder if retaining the concept of an addon here is warranted? Maybe just `custom_iam_policy` is good here. We can then (at last) rid ourselves of the "addons" concept or recycle it to mean something else.